### PR TITLE
replaces `get_headers` $associative with bool

### LINKED ISSUE #8125: `get_headers` `$associative` changed type on php8
# Feature Request

Thank you very much for making rector. It simplifies my work :heart: 
 
I'm currently upgrading a legacy project to php8. I stumbled upon a use of `get_headers`, which rector isn't aware of it's changes parameter `$associative`.

https://www.php.net/manual/en/function.get-headers.php#refsect1-function.get-headers-changelog
https://getrector.com/demo/06beb867-accf-4a43-a749-442098f0938b

## Diff 

[](https://getrector.com/demo/06beb867-accf-4a43-a749-442098f0938b)

```diff
-get_headers($url, 1);
+get_headers($url, true);
```

### PR DESCRIPTION
**Pull request by mvhirsch**
Closes https://github.com/rectorphp/rector/issues/8125

### CODE REVIEW

**Comment by samsonasik** [comment]
Could you rebase and fix conflict?

**Comment by mvhirsch** [comment]
Rebased :ok_hand:

**Review by samsonasik — APPROVED** [review]

File: config/set/php80.php [hunk 1]
```diff
@@ -100,4 +100,10 @@
             new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'l', 'lt'),
             new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'n', 'ne'),
         ]);
+
+    $rectorConfig
+        ->ruleWithConfiguration(FunctionArgumentDefaultValueReplacerRector::class, [
```
Comment (samsonasik) [inline config/set/php80.php]: This can be merged with existing `ruleWithConfiguration(FunctionArgumentDefaultValueReplacerRector::clas` above it

**Comment by samsonasik** [comment]
Thank you @mvhirsch

**Pull request merged**